migration/actual_migrations_test.go

```diff
@@ -758,3 +758,30 @@ func TestMigration24(t *testing.T) {
 	)
 	helpers.FailOnError(t, err)
 }
+
+func TestMigration25(t *testing.T) {
+	db, dbDriver, closer := prepareDBAndInfo(t)
+	defer closer()
+
+	if dbDriver == types.DBDriverSQLite3 {
+		// sqlite is no longer supported
+		return
+	}
+
+	err := migration.SetDBVersion(db, dbDriver, 24)
+	helpers.FailOnError(t, err)
+
+	_, err = db.Exec(
+		`SELECT impacted_since FROM recommendation`,
+	)
+	assert.NotNil(t, err)
```

Comment:
use `assert.NoError` instead please

Reply:
I actually want to assert that an error did in fact occur